javascript/ql/test/query-tests/Security/CWE-079/ReflectedXssWithCustomSanitizer_old.expected

```diff
@@ -1,13 +1,13 @@
+WARNING: Predicate flowsFrom has been deprecated and may be removed in future (ReflectedXssWithCustomSanitizer_old.ql:21,11-20)
```

Comment:
Jenkins failure: you need to use a different version of `/code` to get the samme warning order.